modules/web/js/ballerina/model/default-node-factory.js

```diff
@@ -551,7 +551,7 @@ class DefaultNodeFactory {
         if (functionDef.getParameters()) {
            let parameters = [];
            functionDef.getParameters().map((param) => {
-               let defaultValue = Environment.getDefaultValue(param.type) || null;
+               let defaultValue = Environment.getDefaultValue(param.type) || 'null';
```

Comment:
any specific reason for having null as a string?

Reply:
This is to add null as the parameter. We have to set it here. Maybe we can change the getDefaultValue to check and return null in order to avoid redundant logic. @NatashaWso2 shall we do that?

Reply:
The getDefaultValue() checks and returns null if the default value is undefined or nil